ConwaysGameOfLife.Tests/UniverseMemberData.cs

```diff
@@ -17,5 +18,103 @@ public static class UniverseMemberData
                 new Cell(CellState.Dead, 2, 1),
                 new Cell(CellState.Dead, 2, 2)
             };
+
+        public static TheoryData<string[]> UniversesFromStrings =
+            new TheoryData<string[]> 

```

Comment:
```suggestion
            new TheoryData<object[]> 

```
either that or an array of string arrays